### DISCUSSION TITLE
Question about the player

### DISCUSSION THREAD

**Discussion by TheGlitch1** [tsaridas/stremio-docker#30] [General]
Hi there , i just started experiementing your stremio docker and i really like it . 

I  have one question reagarding the server player ,which is hooked to port 11470. is there any way so i can change the streaming player and set it to my_adress:11470 . whitout always change it from the setting in the stremio app ?

**Comment by rain2307** [comment]
Did you set the `volumes/v` ? then you can save your settings.
